agent.go

```diff
@@ -181,7 +181,7 @@ var logsVSockPort = uint32(0)
 var debugConsoleVSockPort = uint32(0)
 
 // Timeout waiting for a device to be hotplugged
-var hotplugTimeout = 3 * time.Second
+var hotplugTimeout = 10 * time.Second
```

Comment:
some container engines like docker and podman kill the container if its status has not changed to `started` after 10s - so the kata container will be killed before timing out - did you consider this?

Reply:
Well, in that case the higher level engine could time out before us, but that's kind of unavoidable.  In practice the total delay with SHPC is usualy ~6s, so we'd probably complete in time.  The 10s here is just on the basis that the timeout should generally be substantially longer than the actual expected time, to avoid false positives.